internal/graph/graph.go

```diff
@@ -97,3 +97,14 @@ func addCallToResponseMetadata(metadata *v1.ResponseMeta) *v1.ResponseMeta {
 		DebugInfo:           metadata.DebugInfo,
 	}
 }
+
+func addAmortizedCallToResponseMetadata(metadata *v1.ResponseMeta) *v1.ResponseMeta {
```

Comment:
This function doesn't actually add any calls to anything, amortized or not. I think you were trying to explain why we're _not_ adding it (because it's amortized) but the function should probably just explain what it does, and then you can explain why you're calling it at the callsite.

Reply:
Changed the name